addons/website_crm/tests/test_website_crm.py

```diff
@@ -10,14 +10,14 @@ class TestWebsiteCrm(odoo.tests.HttpCase):
     def test_tour(self):
         # change action to create opportunity
         self.start_tour("/", 'website_crm_pre_tour', login='admin')
-        self.start_tour("/", 'website_crm_tour')
 
-        # check result
-        record = self.env['crm.lead'].search([('description', '=', '### TOUR DATA ###')])
-        self.assertEqual(len(record), 1)
-        self.assertEqual(record.contact_name, 'John Smith')
-        self.assertEqual(record.email_from, 'john@smith.com')
-        self.assertEqual(record.partner_name, 'Odoo S.A.')
+        with odoo.tests.RecordCapturer(self.env['crm.lead'], []) as capt:
+            self.start_tour("/", 'website_crm_tour')
+
+        self.assertEqual(len(capt.records), 1)
+        self.assertEqual(capt.records.contact_name, 'John Smith')
+        self.assertEqual(capt.records.email_from, 'john@smith.com')
+        self.assertEqual(capt.records.partner_name, 'Odoo S.A.')
```

Comment:
```suggestion
        self.assertEqual(len(capt.records), 1)
        self.assertEqual(capt.records.contact_name, 'John Smith')
        self.assertEqual(capt.records.email_from, 'john@smith.com')
        self.assertEqual(capt.records.partner_name, 'Odoo S.A.')
        self.assertEqual(capt.records.description, '### TOUR DATA ###')
```

Reply:
Well the description contains `<p>### TOUR DATA ###</p>`... Leaving it this way 👀